app/controllers/api/v2/ptables_controller.rb

```diff
@@ -52,7 +52,7 @@ def create
         process_response @ptable.save
       end
 
-      api :POST, "/ptables/import", N_("Import a provisioning template")
+      api :POST, "/ptables/import", N_("Import a partition table")
```

Comment:
`Import a partition table template` would be IMHO better since we use word template in all other parameters

Reply:
I'm a big fan of consistency. In listing we have `List all partition tables`:
https://github.com/theforeman/foreman/blob/234d1e80aaafe91622dcf33296ff3d581bca2df1/app/controllers/api/v2/ptables_controller.rb#L12

IMHO it either needs to be `partition table template` everywhere or stick to `partition table`. Given this is making it consistent, I'd consider it a bugfix we can safely cherry pick. We can't do that with a massive rename. So I'd ack this.

Reply:
I'll resolve the conversation if @stejskalleos agrees.